### PR TITLE
Consolidate attributon trigger headers

### PR DESCRIPTION
**Pull request by linnan-github**
#432

### CODE REVIEW

File: AGGREGATE.md [hunk 1]
```diff
@@ -98,39 +98,33 @@ at most 32 digits.
 
 ### Attribution trigger registration
 
-Trigger registration will also include two new headers:
-`Attribution-Reporting-Register-Aggregatable-Trigger-Data` is a list of dict
-which generates aggregation keys.
+Trigger registration will also adds two new fields to the JSON dictionary of the
```
Comment (apasel422) [inline AGGREGATE.md:101]:
```suggestion
Trigger registration will also add two new fields to the JSON dictionary of the
```

File: EVENT.md [hunk 2]
```diff
@@ -516,20 +524,14 @@ fully understood during roll-out and help flush out any bugs (either in browser
 or caller code), and more easily compare the performance to cookie-based
 alternatives.
 
-Source registration will accept a new parameter `debug_key`:
+Source and trigger registrations will both accept a new parameter `debug_key`:
```
Comment (apasel422) [inline EVENT.md:527]:
```suggestion
Source and trigger registrations will both accept a new field `debug_key`:
```

**Review by apasel422 — CHANGES_REQUESTED** [review]

**Comment by apasel422** [comment]
Please also resolve the merge conflicts.

**Comment by linnan-github** [comment]
> Please also resolve the merge conflicts.

Already resolved the conflict, thanks.

**Review by apasel422 — APPROVED** [review]
LGTM but please wait for @johnivdel

File: EVENT.md [hunk 3]
```diff
@@ -611,11 +613,13 @@ including `ad-tech.example`, by adding conversion pixels:
 `toasters.example`. They must compress all of the data into 3 bits, so
 `ad-tech.example` chooses to encode the value as "2" (e.g. some bucketed version
 of the purchase value). They respond to the request with a
```
Comment (apasel422) [inline EVENT.md:615]:
```suggestion
of the purchase value). They respond to the request with an
```

File: AGGREGATE.md [hunk 4]
```diff
@@ -98,39 +98,33 @@ at most 32 digits.
...
-an amount of an abstract "value" to contribute to each key, which can be
-integers in [1, 2^16). These are attached to aggregation keys in the order they
-are generated. See the [contribution
-budgeting](#contribution-bounding-and-budgeting) section for more details on how
-to allocate these contribution values.
+  ... // existing fields, such as `event_trigger_data`
 
-```jsonc
-{
+  "aggregatable_trigger_data": [
```
Comment (johnivdel) [inline AGGREGATE.md:108]: nit: Should we just use "event" and "aggregatable" as the top-level keys?

aggregatable_trigger_data is long and a bit redundant with the header name

**Review by johnivdel — COMMENTED** [review]
lgtm % question

File: AGGREGATE.md (hunk 4 above)
Comment (apasel422) [reply AGGREGATE.md:108]: At some point we discussed `event_level_{cfg|cfgs|configs}` and `aggregatable_{cfg|cfgs|configs}`.

Comment (csharrison) [reply AGGREGATE.md:108]: I like @johnivdel's suggestion in principle, the only problem is that `aggregatable` and `aggregatable_values` living side by side a bit weird. You need some clarification on what the `aggregatable` key is describing that isn't described by `aggregatable_values`.

**Review by csharrison — APPROVED** [review]

File: AGGREGATE.md (hunk 4 above)
Comment (johnivdel) [reply AGGREGATE.md:108]: Yeah that's fair. I guess on the aggregate side what we have looks fine.

The event_trigger_data field is a bit ugly as you have trigger_data defined directly inside it but overall I would prefer the names stay aligned.

**Review by johnivdel — APPROVED** [review]
LGTM

**Pull request merged**